### DISCUSSION TITLE
SPT is still working?

### DISCUSSION THREAD

**Discussion by TheMadeX** [synthetichealth/synthea#1528] [Q&A]
Does anybody know if SPT is still working? I installed and tryed to generate a Dockerfile with it using the Guided selection and had troubles all the way, from creating the image untill the generation of patients csv. Can someone point me to a right direction, please? 
Thank you in advance!

**Comment by dehall** [comment]
The Dockerfiles that SPT creates should still work - can you share any more info about the issues you encountered?
